framework/src/main/java/org/checkerframework/common/wholeprograminference/scenelib/ASceneWrapper.java

```diff
@@ -1,19 +1,24 @@
 package org.checkerframework.common.wholeprograminference.scenelib;
 
+import static org.plumelib.util.CollectionsPlume.mapList;
```

Comment:
I think we avoid statically importing methods/fields.

Reply:
If this is not statically imported, then the call is little shorter than the verbose stream version.  I propose to make an exception for this method.